### PR TITLE
Document how to log access to stdout

### PR DESCRIPTION
**Pull request by mogul**
Previously there was no documentation of the fact that `/dev/stdout` was a valid logging target.

### CODE REVIEW

**Review by anita-flegg — CHANGES_REQUESTED** [review]
Hi, Some suggestions/questions:
- This would be clearer with an example. Please add at least one.
- If you can't include the list of values for "format," you at least need to provide a couple of examples of what format values are valid.
- It looks like "Syntax" is part of the command. Is this correct?
- It would be clearer to show file as <file>. This shows that "file" will be replaced by the filename. An example can help show that the filename can include its path (if that is possible).
Thanks

**Comment by mogul** [comment]
> * This would be clearer with an example. Please add at least one.

I refrained from adding one because unlike the `error_log` directive, which can appear at the top level of `nginx.conf`, the specification for the `access_log` directive differs between the NGINX modules that implement it. For example, compare [the `stream` module](https://nginx.org/en/docs/stream/ngx_stream_log_module.html) to [the `http` module](https://nginx.org/en/docs/http/ngx_http_log_module.html). The directive will look different depending on where you're adding it in the `nginx.conf` file.

> * If you can't include the list of values for "format," you at least need to provide a couple of examples of what format values are valid.

Again, the fields available for logging vary for each NGINX module, and are also user-customizable. For example [the specification for `log_format` in the `http` module is pretty large](https://nginx.org/en/docs/http/ngx_http_log_module.html#log_format) and provides an implicit default of `combined` while [the stream version is also complex but there's less available](https://nginx.org/en/docs/stream/ngx_stream_log_module.html#log_format) and the format has to be explicitly specified and referenced. I don't want to just reproduce NGINX documentation here if at all possible. Is it permissible to link out to the nginx.org site?

> * It looks like "Syntax" is part of the command. Is this correct?

I was only following the structure already set by the `error_log` example already in place.

> * It would be clearer to show file as . This shows that "file" will be replaced by the filename. An example can help show that the filename can include its path (if that is possible).

Sorry, I'm not clear what you're asking here. Can you make a comment on the particular line, and use the `+/-` button in GitHub's editor to show me what you think should be different?

In summary: All I really want to get across is that `/dev/stdout` is a valid file target for logging, whether via `error_log` (which is already touched upon) or via `access_log` (which was conspicuously left out). If you can think of a more compact way to express that, then I'll happily take that and rescind this PR!

File: nginx/index.html.md.erb [hunk 1]
```diff
@@ -132,11 +132,26 @@ To load a built-in module like `ngx_stream_module`, add the line below to the to
...
-By default, debug logging will be deactivated in the NGINX buildpack. This helps optimize performance. Set the debug level with the `error_log` directive in the `nginx.conf` to enable debug logging.
+By default, logging is deactivated in the NGINX buildpack. This helps optimize performance. 
 
-To use the debug logging, add the `error_log` entry to your `nginx.conf` file, using the syntax below:
+#### <a id='enabling-access-logs'></a> Enabling access logs
+
+Use the `access_log` directive in the appropriate location in the `nginx.conf` to enable access_logging. Add the `access_log` entry to your `nginx.conf` file using the syntax below:
+
+```
+Syntax:  access_log file [format]
```
Comment (anita-flegg) [inline nginx/index.html.md.erb:144]: On this line, I was trying to show "file" with angle brackets around it. These seem to have been stripped out.
Anything in code like this should be able to be used directly on the command line. If the word "Syntax" is not part of the command users would enter, then it will cause a problem.

File: nginx/index.html.md.erb [hunk 2]
```diff
@@ -132,11 +132,26 @@ To load a built-in module like `ngx_stream_module`, add the line below to the to
...
+```
+
+Where:
+
+* `file` is the name of the file where the log is to be stored. The special value `/dev/stdout` selects the standard output.
+* `format` is the logging format to use. Refer to NGINX documentation for valid customization options.
```
Comment (anita-flegg) [inline nginx/index.html.md.erb:150]: If you aren't willing to provide an example command or possible values for "format," then there at least has to be a link to where users can find this information. Can you add a link to the relevant page of the NGINX documentation?

File: nginx/index.html.md.erb [hunk 3]
```diff
@@ -132,11 +132,26 @@ To load a built-in module like `ngx_stream_module`, add the line below to the to
...
 <code>load_module some_module_dir/my_module.so</code></p>
 
-### <a id='enabling-debugging'></a> Enabling debugging
+### <a id='enabling-logging'></a> Enabling logging
 
-By default, debug logging will be deactivated in the NGINX buildpack. This helps optimize performance. Set the debug level with the `error_log` directive in the `nginx.conf` to enable debug logging.
+By default, logging is deactivated in the NGINX buildpack. This helps optimize performance. 
 
-To use the debug logging, add the `error_log` entry to your `nginx.conf` file, using the syntax below:
+#### <a id='enabling-access-logs'></a> Enabling access logs
```
Comment (anita-flegg) [inline nginx/index.html.md.erb:139]: If the whole purpose of this section is to show the ability to direct to stdout, then it would be good to show this in the section header and/or the intro lines.

**Comment by mogul** [comment]
Made some updates... Can you take another look?

**Comment by anita-flegg** [comment]
Nice work!
